commitlint.groovy

```diff
@@ -1,10 +1,23 @@
 def cico_retries = 16
 def cico_retry_interval = 60
+def duffy_pool = 'virt-ec2-t2-centos-8s-x86_64'
```

Comment:
@nixpanic from where we get this value options like `virt-ec2-*` ?

Reply:
That is the pool the legacy `cico` command got the systems from. I found this pool in the output of the `duffy list-sessions` command when some of our CI jobs were running. There are many more pools, and they are probably documented somewhere, but this was easier/quicker.